app/views/apps/ep_vote_app/application_forms/identity.html.erb

```diff
@@ -3,37 +3,14 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= form_for @application_form do |f| %>
+    <%= form_for @application_form, :builder => AppFormBuilder do |f| %>
```

Comment:
Rozmyslam ci `AppFormBuilder` nie je prilis genericky nazov. Toto bude asi aj mimo apps pouzivane. Zvazujem ci nepreklopit uplne cely default form builder `ActionView::Base.default_form_builder`.

Reply:
Moze to trochu interferovat s formami v adminovi, ak nahodou pouzivaju nejake fieldy, ktore nie su pretazene v AdminFormBuilderi. Ja som totiz predpokladal, ze option[:label] bud je zadefinovany, alebo sa label nema zobrazit. V adminovi sa vytvaral label z nazvu metody.

Reply:
Rozumiem, nechajme teda takto.

Reply:
Chapem, moze byt.